proposals/76:vaults-auctions/upgradeVaults.test.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-env node */
+import test from 'ava';
+
+import {
+  addPreexistingOracles,
+  agops,
+  ATOM_DENOM,
+  bankSend,
+  createBid,
+  getInstanceBoardId,
+  getISTBalance,
+  getLiveOffers,
+  getPriceQuote,
+  getVaultPrices,
+  openVault,
+  pushPrices,
+  USER1ADDR,
+} from '@agoric/synthetic-chain';
+import { readFile, rm } from 'node:fs/promises';
+import {
+  lastAuctionInstancePathname,
+  getDetailsMatchingVats,
+} from './vatDetails.js';
+
+const oraclesByBrand = new Map();
+
+let roundId = 2;
+
+const setupOracles = async t => {
+  const atomOutPre = await getPriceQuote('ATOM');
+  t.is(atomOutPre, '+12010000');
+
+  await addPreexistingOracles('ATOM', oraclesByBrand);
```

Comment:
`addPreexistingOracles` only uses the 1s arg to do `oraclesByBrand.set('ATOM', ..)`. Odd API. I'd expect the caller to do that.  I guess that pre-dates this PR, so IOU an issue on synthetic chain.